### DISCUSSION TITLE
Path to requirements.yml

### DISCUSSION THREAD

**Discussion by finswimmer** [ansible/ansible-lint#3370] [Q&A]
Hey,

is it somehow possible to tell `ansible-lint` where it can find the `requirements.yml` to install additional collections?

It seems to me, that `ansible-lint` is only looking in the current working directory for it. I like to have all ansible related stuff in a subfolder of my repository. `ansible-lint` runs via `pre-commit`. Unfortunately I cannot tell `pre-commit` to change the folder before running.

Thanks!

fin swimmer

**Comment by gizero** [comment]
Hi! Different scenario but similar need to tell ansible-lint what my requirement file is. In my setup I have different requirements files for different environments. My CICD workflow will pick the relevant to run `ansible-galaxy install -r <requirement-file>`, which is fine except for the task of running `ansible-lint` where I wasn't able to find a way to use names other than `requirements.yml`. From documentation I see this file can be located in a few paths, but the name seems to be hardcoded. Did I miss a configuration bit for this. I can run `ansible-lint` with `--offline`, but AFAIU this will be a sub-optimal approach.
